chat_rooms.md

```diff
@@ -2,23 +2,28 @@
 
 Amoung other communication tools, YunoHost project use chat rooms to communicate.
 
-This instant messaging chat rooms, which uses [XMPP protocol](XMPP_en), are hosted on our XMPP server.
-
-You could join this chat rooms with an [XMPP client](https://en.wikipedia.org/wiki/Comparison_of_instant_messaging_clients#XMPP-related_features).
+You could join those chat rooms using :
+- an [IRC Client](https://en.wikipedia.org/wiki/Comparison_of_Internet_Relay_Chat_clients) for example [kiwiirc](https://kiwiirc.com/client/irc.freenode.net/yunohost)
+- an [XMPP client](https://en.wikipedia.org/wiki/Comparison_of_instant_messaging_clients#XMPP-related_features)
+- a [Matrix client](https://matrix.org/docs/guides/faq.html#what-clients-are-available%3F)
 
 ### Chat rooms
-#### Mutual aid
-[Mutual aid](support_en) chat room is here to allow YunoHost's users to help each other.
+#### Mutual aid and support
+- IRC channel **#yunohost** on irc.freenode.net. Also accessible on Matrix (#freenode_#yunohost:matrix.org)
 
-- A [web client](https://chat.yunohost.org) is here to easily join _mutual aid_ chat room.
-- **[support@conference.yunohost.org](xmpp:support@conference.yunohost.org?join)**
+- XMPP chatroom **[support@conference.yunohost.org](xmpp:support@conference.yunohost.org?join)**
 
 #### Development
 YunoHost core development chat room. Currently, the main chat room for contributions on YunoHost project.
-For help, thanks to do it on `support` chat room.
+For help, thanks to do it on **#yunohost** chat room chat room.
```

Comment:
- "thanks to do it on #yunohost" -> "please see #yunohost " (i know this is from the old but merp :s )
- duplicate "chat room"